project-base/kubernetes/kustomize/overlays/ci/kustomization.yaml

```diff
@@ -0,0 +1,9 @@
+bases:
+-   ../../base
+resources:
+-   ../../../deployments/adminer.yml
+-   ../../../deployments/redis-admin.yml
+-   ../../../deployments/selenium-server.yml
+-   ../../../services/adminer.yml
```

Comment:
and for `base/`
since here we have google cloud we can presume that user will use smtp-server from google so maybe just some configuration change in parameters.yml would be needed

Reply:
I guess so, where else? I sometimes look at adminer.

Lets talk about the base/ thing, i dont know what do you mean.

Reply:
maybe it is possible that we could use gmail services instead of smtp-server container.

Reply:
We don't allow any outside traffic into `adminer` or `redis-admin` in the kubernetes manifests atm, so I think we can skip their deployment altogether (or set up some way of accessing them in CI).